### PR TITLE
Livemap tickets

### PR DESCRIPTION
**Pull request by danleech-defra**
All live map tickets and minor Outlook fixes

Epic
https://eaflood.atlassian.net/browse/FIS-217

### CODE REVIEW

File: server/models/outlook.js [hunk 1]
```diff
@@ -122,13 +119,15 @@ class Outlook {
     */
 
     const issueDate = new Date(outlook.issued_at)
-    const date = issueDate.getDate()
+    // const date = issueDate.getDate() // DL: incorrect dates in model
```
Comment (teddmason) [inline server/models/outlook.js:122]: For lines 122-130. Why are the lines of code commented out? Shouldn't leave unnecessarily commented out code. But it looks like the new lines do pretty much the same as the commented out

File: server/src/js/components/map/layers.js [hunk 2]
```diff
@@ -0,0 +1,230 @@
+'use strict'
```
Comment (teddmason) [inline server/src/js/components/map/layers.js:1]: I appreciate the commented out code is to store the different layers we need to investigate, but may be worth putting on a feature branch.

File: server/src/js/pages/alerts-and-warnings.js [hunk 3]
```diff
@@ -1,44 +1,18 @@
 'use strict'
 import '../components/nunjucks'
-import '../components/maps'
-import '../components/maps-styles'
-import '../components/maps-layers'
-import '../components/map-container'
-import '../components/maps/live'
+import '../components/map/maps'
```
Comment (teddmason) [inline server/src/js/pages/alerts-and-warnings.js:3]: I like the restructuring of the files too components/map/class :+1:

**Comment by danleech-defra** [comment]
Test code, redundant and commented out code removed

**Pull request merged**